pkg/asset/manifests/operators.go

```diff
@@ -110,17 +110,13 @@ func (o *manifests) Generate(dependencies map[asset.Asset]*asset.State) (*asset.
 	state := &asset.State{
 		Contents: []asset.Content{
 			{
-				Name: "cluster-config.yaml",
+				Name: filepath.Join(manifestDir, "cluster-config.yaml"),
 				Data: []byte(kubeSys),
 			},
 			{
-				Name: "tectonic-config.yaml",
+				Name: "tectonic/cluster-config.yaml",
```

Comment:
nit: if feels strange to use `filepath.Join` for `cluster-config.yaml` above and then to use a hard-coded slash here.  Can we use `filepath.Join` everywhere (with a variable to hold `tectonic`?) or drop `manifestDir` and just hard-code that everywhere?

Reply:
Fixed. We will eventually need to figure out how to deal with Windows vs. Linux paths if we are going to support running this on Windows. It's fine for now though.